src/league.ts

```diff
@@ -506,17 +506,19 @@ export class League {
     getTeamByPlayerName(playerName: string) {
         playerName = playerName.toLowerCase()
         const directTeamMapping =
-            this._playerLookup[playerName.toLowerCase()] &&
-            this._playerLookup[playerName.toLowerCase()].team
+            this._playerLookup[playerName] &&
+            this._playerLookup[playerName].team
         if (directTeamMapping) return directTeamMapping
         // Try to find the team by looking through the pairings for this
         // playername.  This will find alternates.
-        const teams: Team[] = this._teams.filter((t) => {
```

Comment:
This didn't actually work I think - looks like Heltour only returns TeamMembers in the players list

Reply:
I think we should consider updating Heltour to include alternates in the team list during the time periods where they are part of a team. However, cyanfish may disagree. So I'm fine with this as a fix.